src/components/performances/PerformancesSection.jsx

```diff
@@ -71,10 +74,24 @@ class PerformancesSection extends React.Component {
     // Filter function that accepts a performance object and returns true
     // if this performance passes the filter (i.e should be included)
     const filterFunction = (performance) => {
-      const danceTitle = performance.danceTitle.toLowerCase();
+      const danceTitle = performance.danceTitle ? performance.danceTitle.toLowerCase() : '';
       // Iterates through each performance metadata/key (i.e. academicLevel) and then
       // iterates through each string in the array (i.e. secondary, primary)
-      const isFilterSuccess = keys.every(key => filtersObj[key].includes(performance[key]));
+      const isFilterSuccess = keys.every((key) => {
+        // awardConsiderations need to be checked differently as it's a Number rather
+        // than a String
+        if (key === 'awardConsideration') {
+          if (filtersObj[key].includes('choreoAward')) {
+            return performance.choreoAwardEnum > 0;
+          }
+          if (filtersObj[key].includes('specialAward')) {
```

Comment:
else if here.

Reply:
@YangStephen  adjudications can have both choreAward and specialAward considerations

Reply:
actually, i guess we're iteration through each key, then yes it should be else if

Reply:
Yeah, please do verify the my logic is correct. I'd assume if the filterObj contains choreoAward, then choreoAward must be ticked. So it makes sense to forcefully return out of this statement since the failure of choreoAward to match here means that the current performance should not be filtered on.

Else if was just to change the if-return, if-return pattern.

Reply:
okay, I was actually confused above or disregard what I said before. After thinking and testing, both logics work (and I can explain that another time). However, I am leaning towards having it the way it is: 

```
if (filtersObj[key].includes('choreoAward')) {
    return performance.choreoAwardEnum > 0;
}
if (filtersObj[key].includes('specialAward')) {
    return performance.specialAwardEnum > 0;
}

return true;
```

why? Because the result is not conditional, this is an example of what `filterObj` looks like with values in `awardConsideration`:

``
{
   awardConsideration : ["choreoAward", "specialAward"]
}
```

so when `key === "awardConsideration`, we could have both, so technically, 
if (.... ) else if (...) else (...) doesn't really match the logic we're trying to portray. B/c you can have a `filterObjc[key]` that has both. Hope this helps! :)

Reply:
Got it, I won't debate too much on this topic but my main reason is that *from a code perspective* the second if cannot execute without the first.